types/src/legacy.ts

```diff
@@ -165,6 +176,17 @@ export const legacyTypes: RegistryTypes = {
   FailedAt,
   IPNSIdentity,
   ServiceProviderRecord,
+  ContentId,
+  ContentParameters,
+  DataObjectStorageRelationship,
+  DataObjectStorageRelationshipId,
+  DataObjectType,
+  DataObjectTypeId,
+  NewAsset,
+  ObjectOwner,
+  StorageObjectOwner,
+  UploadingStatus,
+  VoucherLimit
```

Comment:
```suggestion
  VoucherLimit,
```

To fix linter error.
This would be fixed by running `yarn format` in `types/` directory.